### PR TITLE
adding note to ensure having all servers down

### PR DESCRIPTION
**Pull request by felixboehm**
## Description
Adding note, when enabling maintenance mode.

## Motivation and Context
Shutdown servers is considered as best practice for maintenance.

### CODE REVIEW

File: core/Command/Maintenance/Mode.php [hunk 1]
```diff
@@ -61,6 +61,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		if ($input->getOption('on')) {
 			$this->config->setSystemValue('maintenance', true);
 			$output->writeln('Maintenance mode enabled');
+			$output->writeln('Please also consider to stop the server (apache) on all ownCloud webserver instances');
```
Comment (mmattel) [inline core/Command/Maintenance/Mode.php]: server (apache) --> webserver
makes it neutral

Comment (PVince81) [inline core/Command/Maintenance/Mode.php]: also use the theme's product name instead of "ownCloud"

**Comment by DeepDiver1975** [comment]
> also use the theme's product name instead of "ownCloud"

this is occ - no need for white labeling ... we do this nowhere in occ

**Comment by mmattel** [comment]
:+1: 
Maybe you could add the opposite text at the elseif statement (check that the webservers are enabled) to complete this.

**Pull request merged**